test/test-ipld-dag-pb.js

```diff
@@ -22,12 +22,29 @@ module.exports = (repo) => {
     let cid3
 
     before((done) => {
-      node1 = new dagPB.DAGNode(new Buffer('I am 1'))
-      node2 = new dagPB.DAGNode(new Buffer('I am 2'))
-      node3 = new dagPB.DAGNode(new Buffer('I am 3'))
-
       series([
         (cb) => {
+          dagPB.DAGNode.create(new Buffer('I am 1'), (err, node) => {
+            expect(err).to.not.exist
+            node1 = node
+            cb()
+          })
+        },
+        (cb) => {
+          dagPB.DAGNode.create(new Buffer('I am 2'), (err, node) => {
+            expect(err).to.not.exist
+            node2 = node
+            cb()
+          })
+        },
+        (cb) => {
+          dagPB.DAGNode.create(new Buffer('I am 3'), (err, node) => {
+            expect(err).to.not.exist
+            node3 = node
+            cb()
+          })
+        },
```

Comment:
@VictorBjelkholm true, in an example where the only other thing that happens is an assignment, that would be ok, but if there was more operations between that `expect(err)` and `done`, other operations would fail and the error would have the right information for debugging, which would not get printed due to the operations in between failing. For the sake of consistency (check for the error first thing), I keep doing it this way.